### DISCUSSION TITLE
Difference between Bom and Pom file

### DISCUSSION THREAD

**Discussion by bhuvi11** [DependencyTrack/dependency-track#1263] [Q&A] [answered]
Hello Team,

I am having a maven project and i have defined my dependencies in POM.XML file. 
To use Dependency track i would need to generate a BOM FIle. I have done that using Cyclonedx maven plugin.
What is the difference between BOM and POM File?

**Comment by stevespringett** [comment, marked as answer]
A Maven pom defines the project structure including the stated dependencies.
https://maven.apache.org/guides/introduction/introduction-to-the-pom.html

A bom defines the complete bill of materials of what dependencies are actually used - the effective dependencies. This is usually very different than the dependencies listed in the pom, as Maven has a dependency resolution mechanism that turns the dependencies stated in the pom, into effective dependencies, which are the dependencies that are actually resolved. Often times the version of what is effective will be different than what is stated, and the dependency tree will be much more verbose than what was stated. The bom will also give you a lot more data about the dependencies than the pom is capable of representing.
